Flick/Views/PurpleRoundButton.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  PurpleRoundButton.swift
+//  Flick
+//
+//  Created by Haiying W on 12/25/20.
+//  Copyright © 2020 flick. All rights reserved.
+//
+
+import UIKit
+
+class PurpleRoundButton: UIButton {
```

Comment:
Is purple button kinda the same as gray button but just diff color scheme? just wondering if we can combine the two

Reply:
haha you're right it can def be combined